reflex/utils/prerequisites.py

```diff
@@ -265,75 +266,73 @@ def download_and_run(url: str, *args, show_status: bool = False, **env):
     show(f"Installing {url}", process)
 
 
-def download_and_extract_fnm_zip(url: str):
+def download_and_extract_fnm_zip():
     """Download and run a script.
 
-    Args:
-        url: The url of the fnm release zip binary.
-
     Raises:
         Exit: If an error occurs while downloading or extracting the FNM zip.
     """
-    # TODO: make this OS agnostic
     # Download the zip file
+    url = constants.FNM_INSTALL_URL
     console.debug(f"Downloading {url}")
-    fnm_zip_file = f"{constants.FNM_DIR}\\fnm_windows.zip"
-    # Function to download and extract the FNM zip release
+    fnm_zip_file = os.sep.join([constants.FNM_DIR, f"{constants.FNM_FILENAME}.zip"])
```

Comment:
Why not [os.path.join](https://docs.python.org/3/library/os.path.html#os.path.join)?  That's what we are joining right? (paths)